watertap/flowsheets/full_water_resource_recovery_facility/BSM2_P_extension.py

```diff
@@ -113,9 +119,10 @@ def main(bio_P=False):
     results = solve(m)
 
     # Switch to fixed KLa in R5, R6, and R7 (S_O concentration is controlled in R5)
-    m.fs.R5.KLa.fix(240)
-    m.fs.R6.KLa.fix(240)
-    m.fs.R7.KLa.fix(84)
+    # KLa for R5 and R6 taken from [1], and KLa for R7 taken from [2]
+    m.fs.R5.KLa.fix(240 / 24)
+    m.fs.R6.KLa.fix(240 / 24)
+    m.fs.R7.KLa.fix(84 / 24)
```

Comment:
Can you update these in our other flowsheets?

Reply:
Will do